packages/compass/src/app/components/home.tsx

```diff
@@ -286,7 +312,7 @@ export function ThemedHome(
                 )}
                 data-theme={darkMode ? 'Dark' : 'Light'}
               >
-                <Home {...props}></Home>
+                <HomeWithConnections {...props}></HomeWithConnections>
```

Comment:
Hierarchy is now ThemedHome -> HomeWithConnections -> Home whereas it was HomeWithConnections -> ThemedHome -> Home.

This allows CompassConnections to be inside all the visual stuff but still wrap the rest of home and therefore the useSingleConnectionModeConnectionInfoStatus() hook still works for now.